presto-orc/src/main/java/com/facebook/presto/orc/stream/LongInputStreamV2.java

```diff
@@ -134,6 +135,7 @@ public int scan(
                 scanLiterals();
             }
         }
+        // TODO: should we skip to endOffset - currentRunOffset + numLiterals?
```

Comment:
Is this still a TODO? Did you find an answer?

Reply:
Yes, Orri confirmed we should be skipping, updating shortly.